WooCommerce/Classes/Authentication/Epilogue/SwitchStoreUseCase.swift

```diff
@@ -4,6 +4,10 @@ import protocol Storage.StorageManagerType
 
 protocol SwitchStoreUseCaseProtocol {
     func switchStore(with storeID: Int64, onCompletion: @escaping (Bool) -> Void)
+
+    /// Returns available WooCommerce stores from storage
+    ///
+    func getAvailableStores() -> [Site]
```

Comment:
💭 this feels like not directly related to the "switch store" use case 🤔 in order to find out the number of stores, how about using `StorePickerViewModel`, calling its `refreshSites`, and checking its `StorePickerState` on completion? the state should inform us whether the stores are empty/1/more.

Reply:
Thanks for your input, @jaclync !

> this feels like not directly related to the "switch store" use case 

I added `getAvailableStores` method to switch store use case because "switching store" also needs knowledge about "available stores". And, it made sense to expose the available stores using a method. 

> how about using StorePickerViewModel, calling its refreshSites, and checking its StorePickerState on completion? the state should inform us whether the stores are empty/1/more.

I hope you meant creating a `StorePickerViewModel` like in `StoreCreationCoordinator`? i.e. A standalone view model instance instead of trying to access the view model from the store picker view controller. 
I will look into this and follow this implementation. 👍

Reply:
> > this feels like not directly related to the "switch store" use case
> 
> I added `getAvailableStores` method to switch store use case because "switching store" also needs knowledge about "available stores". And, it made sense to expose the available stores using a method.

that's true for the current implementation, but the number of available stores doesn't feel like necessary knowledge for switching a store. a different implementation can just log the user into the target store without knowing the number of stores.

> > how about using StorePickerViewModel, calling its refreshSites, and checking its StorePickerState on completion? the state should inform us whether the stores are empty/1/more.
> 
> I hope you meant creating a `StorePickerViewModel` like in `StoreCreationCoordinator`? i.e. A standalone view model instance instead of trying to access the view model from the store picker view controller.

yea I meant just instantiating `StorePickerViewModel` in place and then calling the necessary function to get the number of stores. perhaps this functionality can be refactored to a separate class for getting the latest number of stores in the future.

Reply:
Done in 4aa04057c7c6ca5346c1a99e756a58641c027fd6